apps/tx-builder/src/components/modals/WrongChainBatchModal.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, GenericModal, Text } from '@gnosis.pm/safe-react-components';
+import Box from '@material-ui/core/Box';
+import styled from 'styled-components';
+
+type WrongChainBatchModalProps = {
+  onClick: () => void;
+  onClose: () => void;
+};
+
+const WrongChainBatchModal = ({ onClick, onClose }: WrongChainBatchModalProps) => {
+  return (
+    <GenericModal
+      title="Warning!"
+      withoutBodyPadding
+      body={
+        <StyledModalBodyWrapper>
+          <Text size={'xl'} center>
+            This Batch is from another Chain!
```

Comment:
I'd change the working to something like, "This batch was created on a different network. Please proceed with caution and make sure the contracts exist on the current network". You can also ask a native speaker :D